lib/SILOptimizer/Transforms/PruneVTables.cpp

```diff
@@ -27,10 +27,16 @@ using namespace swift;
 
 namespace {
 class PruneVTables : public SILModuleTransform {
-  void runOnVTable(SILModule *M,
-                   SILVTable *vtable) {
+  void runOnVTable(SILModule *M, SILVTable *vtable) {
     LLVM_DEBUG(llvm::dbgs() << "PruneVTables inspecting table:\n";
                vtable->print(llvm::dbgs()));
+    if (!M->isWholeModule() &&
+        !vtable->getClass()->isOutermostPrivateOrFilePrivateScope()) {
```

Comment:
I would prefer `getClass()->getEffectiveLinkage() < AccessLevel::Internal`

Reply:
Sorry, getEffectiveAccess()

Reply:
I like that better as well.